CalcualtorLogic.js

```diff
@@ -103,8 +105,10 @@ class Calculator {
     updateResultElement() {
         this.resultElement.textContent = this.currentResult;
     }
-    onOperatorButtonClick() {
-
+    onOperatorButtonClick(event) {
+        const curntRsltAndOperclkdBtn = this.currentResult + event.target.dataset.operator;
```

Comment:
Can you explain line by line

Reply:
108 = names of the method and its parameter
109= defining a variable curntRsltAndOperclkdBtn
and assigning its value to current result value added to the clicked operator 
110= 
assigning the current result to the curntRsltAndOperclkdBtn
111 = calling updateResultElement method

Reply:
108 = names of the method and its parameter
109= defining a variable curntRsltAndOperclkdBtn
and assigning its value to current result value added to the clicked operator 
110= 
assigning the current result to the curntRsltAndOperclkdBtn
111 = calling updateResultElement method

Reply:
what do you need `currentResult` here?
how `currentResult` is helping you here?

also, how `updateResultElement ` is helping you?

what you are trying to do by calling `updateResultElement ` here?
Please don't remove the code before answering these questions

Reply:
They will help me if I want to see the clicked operator before adding the second operand which is not necessary

Reply:
In the previous implantation you did something similar and I TOLD YOU to use `console.log` when you want you to know something.
whay did you repeat the same thing here?

Please remove all the method content and re-do it again 
Please read the requirements of the methods again 
if you want to inspect a property value, use console.log and log the value to the console.

Thanks

Reply:
I some the last changes before seeing your last to comments 
but I did not pushed them because you ordered that I have to answer your question first